packages/@aws-cdk/integ-tests-alpha/lib/assertions/providers/provider.ts

```diff
@@ -118,6 +131,13 @@ interface SingletonFunctionProps extends LambdaFunctionProviderProps {
    * We recommend generating a UUID per provider.
    */
   readonly uuid: string;
+
+  /**
+   * How long, in days, the log contents will be retained.
+   *
+   * @default - no retention days specified
+   */
+  readonly logRetention?: RetentionDays;
```

Comment:
to confirm do we need this here if its already defined in `LambdaFunctionProviderProps`

Reply:
They're duplicated, removing this one.